zeus_tracker/plugin.spec.yaml

```diff
@@ -3,14 +3,15 @@ extension: plugin
 product: ["insightconnect"]
 name: zeus_tracker
 title: ZeuS Tracker
-description: ZeuS Command&Control Server Tracking
+description: The ZeuS Command&Control Server Tracking plugin
 version: 1.0.0
 vendor: rapid7
 support: community
-status: ["obsolete"]
+status: [deprecated]
```

Comment:
This plugin is obsolete, will need to revert this back

# ZeuS Tracker has been discontinued on Jul 8th, 2019